### PR TITLE
Issue #239 Enabling support for HyperV

### PR DESCRIPTION
**Pull request by hferentschik**
Fix for issue #239

### CODE REVIEW

File: lib/landrush/action/common.rb [hunk 1]
```diff
@@ -6,6 +6,7 @@ module Common
         'VagrantPlugins::ProviderLibvirt::Provider'    => :libvirt,
         'HashiCorp::VagrantVMwarefusion::Provider'     => :vmware_fusion,
         'VagrantPlugins::Parallels::Provider'          => :parallels,
+        'VagrantPlugins::HyperV::Provider'             => :hyperv,
```
Comment (LalatenduMohanty) [inline lib/landrush/action/common.rb]: +1

File: lib/landrush/plugin.rb [hunk 2]
```diff
@@ -21,6 +21,9 @@ class Plugin < Vagrant.plugin('2')
       hook.before(VagrantPlugins::ProviderVirtualBox::Action::Network, pre_boot_actions)
       hook.after(Vagrant::Action::Builtin::WaitForCommunicator, post_boot_actions)
 
+      hook.before(VagrantPlugins::HyperV::Action::WaitForIPAddress, pre_boot_actions)
+      hook.after(Vagrant::Action::Builtin::WaitForCommunicator, post_boot_actions)
+
```
Comment (LalatenduMohanty) [inline lib/landrush/plugin.rb]: +1

File: lib/landrush/action/common.rb (hunk 1 above)
Comment (hferentschik) [inline lib/landrush/action/common.rb]: thanks :-)
I am going to cut a 1.1.1, which will work for virtualbox, libvirt, hyperv

**Comment by hferentschik** [comment]
Rebased and merged

**Pull request closed without merging**